src/test/js/uplift_user_test.js

```diff
@@ -42,25 +42,38 @@ return Promise.all([
     ]);
 });
 
- Scenario('@functional @uplift I am able to use a pin to create an account as an uplift user', async (I) => {
-     I.amOnPage(TestData.WEB_PUBLIC_URL + '/login/uplift?client_id=' + serviceName + '&redirect_uri=' + redirectUri + '&jwt=' + accessToken);
-     I.waitForText('Create an account or sign in', 30, 'h1');
-     I.fillField('#firstName', randomUserFirstName);
-     I.fillField('#lastName', randomUserLastName);
-     I.fillField('#username', citizenEmail);
-     I.scrollPageToBottom();
-     I.click('Continue');
-     I.waitForText('Check your email', 20, 'h1');
-     var url = await I.extractUrl(citizenEmail);
-     if (url) {
-        url = url.replace('https://idam-web-public.aat.platform.hmcts.net', TestData.WEB_PUBLIC_URL);
-     }
-     I.amOnPage(url);
-     I.waitForText('Create a password', 20, 'h1');
-     I.fillField('#password1', password);
-     I.fillField('#password2', password);
-     I.click('Continue');
-     I.waitForText('Account created', 60, 'h1');
-     I.see('You can now sign in to your account.');
+Scenario('@functional @uplift I am able to use a pin to create an account as an uplift user', async (I) => {
+    I.amOnPage(TestData.WEB_PUBLIC_URL + '/login/uplift?client_id=' + serviceName + '&redirect_uri=' + redirectUri + '&jwt=' + accessToken);
+    I.waitForText('Create an account or sign in', 30, 'h1');
+    I.click("Continue");
+    I.waitForText('Information is missing or invalid', 20, 'h2');
+    I.see('You have not entered your first name');
+    I.see('You have not entered your last name');
+    I.see('You have not entered your email address');
+    I.fillField('firstName', 'L');
+    I.fillField('lastName', '@@');
+    I.click('Continue');
+    I.wait(5);
+    I.see('Your first name is invalid');
```

Comment:
Probably better to leave the 'success' scenario as it is, and add a 'validation' scenario - the same way that self registration does it. It makes it much clearer when people are investigating failures in tests

Reply:
But there isn't an existing one, so I'll leave it up to you

Reply:
agree with james on this one, if we move the validation test to a different method and that fails it's gonna be super obvious it's the validation that is failing and not the uplift itself. But I'm sure I made something similar before. 🤣